### PR TITLE
Displays last modification date and source (optional) of layer from metadata

### LINKED ISSUE #362: Intégration automatique de la date de mise à jour des couches
Dans la légende de l'interface, en plus des sources de données, il serait intéressant d'afficher la date de mise à jour de la donnée de manière automatique. Cela devrait être possible pour toute couche configurée avec une URL de fiche de métadonnée XML conforme ISO 19139 et correctement remplie. 
L’affichage de cette information pourrait être différé, étant donné que la récupération et l’analyse de la métadonnée sont des opérations coûteuses en temps.

layer_date

### PR DESCRIPTION
**Pull request by tkohr**
PR should resolve #362 by getting date of creation or last modification from metadata and displaying it in layer info following legend.

The "source" info from metadata can also be retrieved as attribution by setting `attribution="metadata"` in config.xml.

### CODE REVIEW

File: js/mviewer.js [hunk 1]
```diff
@@ -607,8 +607,13 @@ mviewer = (function () {
                         summary += '<a href="'+_overLayers[this.layer].metadata+'" target="_blank">En savoir plus</a>';
                     }
                     _overLayers[this.layer].summary = summary;
+
+                    var modifiedDate = $(result).find("dct\\:modified, modified").text() || $(result).find("dct\\:created, created").text();
```
Comment (fvanderbiest) [inline js/mviewer.js:611]: XPath to be double checked ...

Comment (tkohr) [reply js/mviewer.js:611]: Looking here https://www.dublincore.org/specifications/dublin-core/dcmi-terms/ and here https://www.w3.org/TR/vocab-dcat-2/#Property:resource_update_date seem to be the correct fields (as far as I understand)

Comment (fvanderbiest) [reply js/mviewer.js:611]: Oh, you're requesting the result as dublin core, not ISO ? Then it's fine.
Could you paste an example of MD URL which gets queried ?

Comment (tkohr) [reply js/mviewer.js:611]: Yep, here's the record for the lycée layer of the default.xml https://kartenn.region-bretagne.fr/geonetwork/srv/eng/csw?SERVICE=CSW&VERSION=2.0.2&REQUEST=GetRecordById&elementSetName=full&ID=99e78163-ce9a-4eee-9ea0-36afc2a53d25

**Pull request merged**